deploy/template.yaml

```diff
@@ -197,77 +201,6 @@ Resources:
       TimeoutInMinutes: 480
       Source:
         Type: "NO_SOURCE"
```

Comment:
How does the BuildSpec work now if it's no longer referenced here?

Reply:
The build specification for the project. If this value is not provided, then the source code must contain a buildspec file named buildspec.yml at the root level. If this value is provided, it can be either a single string containing the entire build specification, or the path to an alternate buildspec file relative to the value of the built-in environment variable CODEBUILD_SRC_DIR. The alternate buildspec file can have a name other than buildspec.yml, for example myspec.yml or build_spec_qa.yml or similar. For more information, see the [Build Spec Reference](https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#build-spec-ref-example) in the AWS CodeBuild User Guide.

Reply:
@Tom-Dann can you put together a doc on how this buildspec will be governed, and what parameters need to be in place to ensure they don't do things they shouldn't - plus do the things that are required (e.g. mk test report .etc) - or just to document that teams should follow and copy/paste your example, then how would they keep checking back in for updates?

Reply:
The BuildSpec governance remains the same, it is just now in it's own file which helps with linting it as native YAML rather than a multi-line string in the context of the larger CloudFormation template